pipeline/plugins/operators/custom_operators.py

```diff
@@ -78,6 +78,7 @@ def __init__(
         self.tmp_download_directory = tmp_download_directory
         self.tar_directory = tar_directory
         self.api = api
+        self.mark_failed = False
```

Comment:
Per instance is fine: Airflow creates a new instance of the class for each (re)run of an operator. This can be seen working as intended e.g. in [these logs for 361_1 batch 13 download](http://128.214.252.206/dags/image_download_col-361/grid?dag_run_id=scheduled__2023-05-22T00%3A00%3A00%2B00%3A00&task_id=download_set.download_image_col-361_1.download_binding_batch&tab=mapped_tasks&map_index=13). The first two attempts have failed as expected, but on the third run we still get errors but the task is marked as success anyway:
```
[2023-10-10, 13:06:21 UTC] {file_download_operators.py:161} ERROR - METS download https://digi.kansalliskirjasto.fi/sanomalehti/binding/1906699 failed with 500, will retry and/or continue with others
[2023-10-10, 13:06:21 UTC] {custom_operators.py:164} ERROR - Downloading METS in https://digi.kansalliskirjasto.fi/sanomalehti/binding/1906699 still failing, moving on with image creation
[2023-10-10, 13:06:22 UTC] {custom_operators.py:228} ERROR - Failed to create tar file for batch 13 from tmp to destination failed
[2023-10-10, 13:06:22 UTC] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=image_download_col-361, task_id=download_set.download_image_col-361_1.download_binding_batch, map_index=13, execution_date=20230522T000000, start_date=20231010T130606, end_date=20231010T130622
[2023-10-10, 13:06:22 UTC] {local_task_job_runner.py:228} INFO - Task exited with return code 0
```